src/entry-points/local-file-player/App.svelte

```diff
@@ -65,6 +64,17 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     videoEl.focus();
   }
 
+  async function onDeleteFile(ind: number) {
+    files = files.filter((file,fileIndex) => fileIndex !== ind)
+    if (ind<currFileInd){
+      // target file comes before current file
+      currFileInd -= 1  // move forward the current index
+    } 
+    // target file comes after current file / target file is current file
+    // then no need to change the current file index
+    playFile(currFileInd); 
```

Comment:
I'm sorry, but doesn't this restart the playback every time you delete a file? Have you actually tested your changes? I think it's gonna be frustrating for the users. Also what if the new length is 0?